android/src/main/java/com/twiliorn/library/CustomTwilioVideoView.java

```diff
@@ -450,7 +456,7 @@ private void setAudioFocus(boolean focus) {
              * speaker mode if this is not set.
              */
             audioManager.setMode(AudioManager.MODE_IN_COMMUNICATION);
-            audioManager.setSpeakerphoneOn(!audioManager.isWiredHeadsetOn());
+            audioManager.setSpeakerphoneOn(false);
```

Comment:
What if wired headset is connected?

Reply:
Bluetooth needs speakerPhone Off settings to function and if it cant find a bluetooth the system defaults to speakerphone anyway.